src/lib/const.ts

```diff
@@ -142,52 +160,52 @@ import "@fluencelabs/aqua-lib/builtin.aqua"
 
 -- import App from "deployed.app.aqua"
 -- import Adder from "services/adder.aqua"
--- export App, add_one
+-- export App, addOne
```

Comment:
If you are doing snake_case to camelCase renaming only for the sake of making generated TS files look nice than I think we are doing it in the wrong place. This should be done inside aqua compiler.

Reply:
@DieMyst WDYT?

@folex WDYT about recommended naming notation?

Reply:
I like snake case. But I don't care much.

Reply:
I think compiler shouldn't do that. It's just camelCase is a lot more popular among js developers so I think people who compile to js would prefer using camelCase themselves

Reply:
Our built-ins library is using snake case in function naming. So if a developer would be using camelCase they would inevitably face style inconsistencies